### ISSUE TITLE
Add cookies for session management.

### ISSUE THREAD

**Issue by geekyharsh05** [yesahem/Medium-Blog#45]
- refresh token
- access token

Add these two things mentioned above.

**Comment by pandeyashu08** [comment]
@geekyharsh05 Can you please assign me

**Comment by yesahem** [comment]
any update @pandeyashu08 ??

**Comment by pandeyashu08** [comment]
@yesahem Sorry i was busy with something else ,  I will do a PR soon.

**Comment by yesahem** [comment]
sure !

**Comment by pandeyashu08** [comment]
@geekyharsh05 My backend is working fine with accesstoken and refreshtoken. but it is a bit difficult to implement it in the frontend. Backend part is done. In frontend we require to have either a context provider or a state management library for this to persist the user is logged in or not.

**Comment by geekyharsh05** [comment]
@pandeyashu08 Yeah I'm getting that. So in that case jwt is fine for now. Keeping it simple.
